content/doc/book/security/access-control/permissions.adoc

```diff
@@ -107,6 +105,101 @@ Anything they cannot accomplish through the existing UI they can do through the
 An instance on which an untrusted user gained _Overall/Administer_ permission should be considered fully compromised and should be replaced.
 All secrets (credentials, etc.) stored on that instance should be rotated or revoked and all artifacts created from that point on should be verified.
 
+
+=== Agent permissions
+
+==== Agent/Build
+
+This permission allows users to run jobs as them on agents.
+
+==== Agent/Configure
+
+This permission allows users to configure agents.
+
+==== Agent/Connect
+
+This permission allows users to connect agents or mark agents as online.
+
+This permission is implied by Agent/Disconnect.
+
+==== Agent/Create
+
+This permission allows users to create agents.
+
+==== Agent/Delete
+
+This permission allows users to delete existing agents.
+
+==== Agent/Disconnect
+
+This permission allows users to disconnect agents or mark agents as temporarily offline.
```

Comment:
Probably makes more sense as a list (e.g. definition list?), given the current lack of details? (Also applies to all other lists at this level.)